src/desktop/apps/fair_organizer/routes.coffee

```diff
@@ -85,5 +88,4 @@ representation = (fair) ->
         res.locals.pastFairs = res.locals.sd.PAST_FAIRS = fairs.pastYearRoundFairs()
         res.locals.articles = articles.models
         next()
-      ).done()
-
+      )
```

Comment:
I'm curious about why we're dropping the `done()`.

Reply:
It's mainly because I want to [return a chain-able promise](http://documentup.com/kriskowal/q/#the-end) that we can [chain in the test](https://github.com/artsy/force/pull/3780/files#diff-b49e7ef5841f8665f1be07134b971029R87).